haskell-language-server.cabal

```diff
@@ -303,264 +302,273 @@ common qualifyImportedNames
-                  , refineImports
-                  , rename
-                  , retrie
-                  , tactic
-                  , hlint
-                  , stan
-                  , moduleName
-                  , pragmas
-                  , splice
-                  , alternateNumberFormat
-                  , qualifyImportedNames
-                  , codeRange
-                  , gadt
-                  , explicitFixity
-                  , explicitFields
-                  , floskell
-                  , fourmolu
-                  , ormolu
-                  , stylishHaskell
-                  , refactor
+  import:
+    common-deps
+    -- configuration
+    , warnings
+    , pedantic
+    -- plugins
+    , cabal
+    , callHierarchy
+    , cabalfmt
+    , changeTypeSignature
+    , class
+    , haddockComments
+    , eval
+    , importLens
+    , refineImports
+    , rename
+    , retrie
+    , tactic
+    , hlint
+    , stan
+    , moduleName
+    , pragmas
+    , splice
+    , alternateNumberFormat
+    , qualifyImportedNames
+    , codeRange
+    , gadt
+    , explicitFixity
+    , explicitFields
+    , floskell
+    , fourmolu
+    , ormolu
+    , stylishHaskell
+    , refactor
 
   exposed-modules:
+    HlsPlugins
     Ide.Arguments
     Ide.Main
     Ide.Version
-    HlsPlugins
 
-  other-modules:    Paths_haskell_language_server
-  autogen-modules:  Paths_haskell_language_server
-  hs-source-dirs:   src
+  other-modules: Paths_haskell_language_server
+  autogen-modules: Paths_haskell_language_server
+  hs-source-dirs: src
   build-depends:
-    , async
-    , base16-bytestring
-    , bytestring
-    , containers
-    , cryptohash-sha1
-    , data-default
-    , ghc
-    , ghcide                ^>=1.9
-    , githash               >=0.1.6.1
-    , lsp
-    , hie-bios
-    , hiedb
-    , hls-plugin-api        ^>=1.6
-    , optparse-applicative
-    , optparse-simple
-    , process
-    , hls-graph
-    , safe-exceptions
-    , sqlite-simple
-    , unordered-containers
-    , aeson-pretty
+    aeson-pretty,
+    async,
+    base16-bytestring,
+    bytestring,
+    containers,
+    cryptohash-sha1,
+    data-default,
+    ghc,
+    ghcide ^>=1.9,
+    githash >=0.1.6.1,
+    hie-bios,
+    hiedb,
+    hls-graph,
+    hls-plugin-api ^>=1.6,
+    lsp,
+    optparse-applicative,
+    optparse-simple,
+    process,
+    safe-exceptions,
+    sqlite-simple,
+    unordered-containers,
 
   default-language: Haskell2010
-  default-extensions: DataKinds, TypeOperators
+  default-extensions:
+    DataKinds
+    TypeOperators
 
 executable haskell-language-server
-  import:           common-deps
-                    -- configuration
-                  , warnings
-                  , pedantic
-  main-is:          Main.hs
-  hs-source-dirs:   exe
-
+  import:
+    common-deps
+    -- configuration
+    , warnings
+    , pedantic
+
+  main-is: Main.hs
+  hs-source-dirs: exe
   ghc-options:
-    -threaded
     -- allow user RTS overrides
```

Comment:
bad comment placement

Reply:
I think it's not a blocker, we can move these comments above the field and clarify